index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;
+    0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;
+    1,700;1,800&display=swap" rel="stylesheet">
+    <title>Nyalik Portfolio</title>
+</head>
+<body>
+    <nav>
+        <ul class="toolbar-container">
+            <li class="logo">Nyalik Marv</li>
+            <li>
+                <svg width="30" height="20" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
+                </svg>    
+            </li>
+        </ul>
+    </nav>
+
+    <main>
+        <section class="headline">
+        <h1 class="heading">I’m Marvin. Glad to see you!</h1>
+
+        <article class="bio">
+            <p> I’m a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+               don’t hesitate to contact me.</p> <br>
```

Comment:
- [ ] To stick as much possible to your chosen Figma design, I think it would be better if you apply the same `text-color`, `font-size`, `line-height`,... like the one used on Figma on the paragraph description text